pages/authorized_page.py

```diff
@@ -12,19 +12,19 @@ class AuthorizedPage(Page):
 
     url = '/home'
 
-    def label_page_head(self):
+    def label_page_head(self, driver):
         self.context.wait.until(EC.visibility_of_element_located((By.CSS_SELECTOR, "div.aui-page-header-main > h1")))
-        return self.context.driver.find_element_by_css_selector("div.aui-page-header-main > h1")
+        return driver.find_element_by_css_selector("div.aui-page-header-main > h1")
 
-    def get_page_head(self):
-        return self.label_page_head().text
+    def get_page_head(self, driver):
+        return self.label_page_head(driver).text
 
-    def enter_app(self):
-        self.find_enter_btn().click()
+    def enter_app(self, driver):
+        self.find_enter_btn(driver).click()
         self.context.wait.until(lambda driver: driver.find_element_by_id('status_dropdown'))
 
-    def find_enter_btn(self):
-        return self.context.driver.find_element_by_xpath('//a[text()="Launch the web app"]')
+    def find_enter_btn(self, driver):
```

Comment:
Dont sure about theese 'driver' parameters. I think its my old drafts for 2-browsers-work feature ? Are they needed ?

Reply:
I would prefer to switch contest instead of passing browser, user, etc. into each step

Reply:
@Bidonus They are needed as we've discussed